.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: ruby
+cache:
+  directories:
+    - bundle
+
+rvm:
+  - 2.4.1
+
+bundler_args: "--without documentation --path bundle"
+
+before_install:
```

Comment:
Why do you do this? doesn't it happen automatically by default? Just curious

Reply:
I stole the .travis.yml from the Danger repo. I dont know :)

Reply:
No problem, just curious, looks totally good to merge to me